src/components/Sidebar/ActionNode.tsx

```diff
@@ -52,6 +59,17 @@ const ActionNode: FC<ActionNodeProps> = ({
     [changeNodeType]
   );
 
+  const addActionCQL = useCallback(
+    (action: Action, currentNodeKey: string): void => {
+      const cql = createCQL(action, currentNodeKey);
+      convertBasicCQL(cql).then(elm => {
+        // eslint-disable-next-line
```

Comment:
How come this is needed? I don't think I see a lint error when I remove it.

Reply:
Oh yea you're right that can be removed now. Before this was its own method it was using `currentNode.key` not `currentNodeKey`. A check was already done to ensure the key was defined but the anonymous function wasn't smart enough to know that. As such I had to use `currentNode.key!` which gave a lint error